pdns/dnsdist-lua-bindings.cc

```diff
@@ -379,10 +379,17 @@ void setupLuaBindings(LuaContext& luaCtx, bool client)
 
   /* NetmaskGroup */
   luaCtx.writeFunction("newNMG", []() { return NetmaskGroup(); });
-  luaCtx.registerFunction<void(NetmaskGroup::*)(const std::string&mask)>("addMask", [](NetmaskGroup&nmg, const std::string& mask)
+  luaCtx.registerFunction<void(NetmaskGroup::*)(const std::string& mask)>("addMask", [](NetmaskGroup&nmg, const std::string& mask)
                          {
                            nmg.addMask(mask);
                          });
+  luaCtx.registerFunction<void(NetmaskGroup::*)(const NetmaskGroup& otherNMG)>("addMMG", [](NetmaskGroup& nmg, const NetmaskGroup& otherNMG) {
```

Comment:
```suggestion
  luaCtx.registerFunction<void(NetmaskGroup::*)(const NetmaskGroup& otherNMG)>("addNMG", [](NetmaskGroup& nmg, const NetmaskGroup& otherNMG) {
```

Reply:
:facepalm: Thanks! I just fixed it, and added a test in the process :)